DocAI Incubator Tools/best-practices/Key_Value_Pair_Entity_Conversion.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #36.    def entity_data(formField_data: dict, page_number: int):
</pre><p>This whole function should use the entity object instead of a dict</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>